### PR TITLE
RHDEVDOCS-5794: Update example

### PR DESCRIPTION
**Pull request by nebelsprachen**
Version(s):

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/about-must-gather.adoc [hunk 1]
```diff
@@ -27,8 +27,7 @@ For example:
 +
```
Comment (eromanova97) [inline modules/about-must-gather.adoc:27]: This is outside of scope, but if possible, I would replace `For example:` with `.Example command` for the consistency. I would make that change in this whole section if possible. Thank you!

File: modules/about-must-gather.adoc [hunk 2]
```diff
@@ -27,8 +27,7 @@ For example:
...
 [source,terminal,subs="attributes+"]
 ----
-$ oc adm must-gather \
-  --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion}
+$ oc adm must-gather --image=registry.redhat.io/openshift-gitops-1/must-gather-rhel8:v1.10.0
```
Comment (eromanova97) [inline modules/about-must-gather.adoc:30]: At the end of this module, there is another example that I think should be changed:
````
Optionally, you can run the `oc adm must-gather` command in a specific namespace by using the `--run-namespace` option.

For example:
[source,terminal,subs="attributes+"]
----
$ oc adm must-gather --run-namespace <namespace> \
  --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion}
----
````

**Review by eromanova97 — COMMENTED** [review]
The change looks good to me, I have just one suggestion and one observation about another example here. Thank you!

Edit: Also I just noticed the versions also need to be yet clarified to be for `gitops-docs-1.10` and later, thanks!

File: modules/about-must-gather.adoc (hunk 1 above)
Comment (Srivaralakshmi) [reply modules/about-must-gather.adoc:27]: @eromanova97 Thanks for the review comments. I agree.

@joaedwar Please get this addressed.

**Comment by Srivaralakshmi** [comment]
@joaedwar May I know about this PR status? It should not take this long for a closure. If there is anything that you need from me to unblock you, please let me know. The JIRA description and comments section information must help you with the details. If you need more clarification, please reach out to me. 

Also, please update the PR description section. Refer to this example: https://github.com/openshift/openshift-docs/pull/70057

**Pull request closed without merging**